src/test/tests/databases/xdmf.py

```diff
@@ -403,6 +403,18 @@ def test7(datapath):
     DeleteAllPlots()
     CloseDatabase(pjoin(datapath,"singleWedge18.xmf"))
 
+def test8(datapath):
+    TestSection("Curves")
+    OpenDatabase(pjoin(datapath,"curve.xmf"))
+    AddPlot("Curve", "Curve/Node Centered Values")
+    ca = CurveAttributes()
+    ca.showLabels = 0
+    SetPlotOptions(ca)
+    DrawPlots()
+    Test("xdmf_8_01")
```

Comment:
Where is the baseline for this?

Reply:
Right...was going to let the test suite generate it for me instead of possibly creating a subtly different result on my mac. I will go en on poodle though. When I am done, I will check the `I have confirmed new and existing unit tests pass locally with my changes` box in this PR.

Reply:
baseline added now, from poodle.